timesheets/Code.gs

```diff
@@ -0,0 +1,148 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+// GLOBAL VARS
+var HRS_START = 4;
+var HRS_END = 8;
+var HRLY_PAY_COLUMN = 9;
+var APPROVAL_COLUMN = 11;
+var EMAIL_COLUMN = 3;
+
+/**
+* Creates the menu item "Run Functions" so user can run scripts on drop-down.
+*/
+function onOpen(e) {
+    var ui = SpreadsheetApp.getUi();
+    ui.createMenu('Timesheets')
+        .addItem('Calculate Pay', 'calculatePay')
+        .addItem('Add Approval Column', 'addApprovalColumn')
+        .addItem('Send Emails', 'sendEmails')
+        .addToUi();
+}
+
+/**
+* Runs the script "Send Emails". Sends e mails to every employee notifying 
+* them whether or not their timesheet has been approved.
+* Checks + updates "NOTIFIED" status so employers do not send e mails more than once.
+*/
+function sendEmails() {
+    var sheet = SpreadsheetApp.getActiveSheet();
+    var lastRow = sheet.getLastRow();
+    var lastCol = sheet.getLastColumn();
+    // lastCol here is the NOTIFIED column
+  
+    // goes through the whole email column
+    for (i =2; i <= lastRow; i++) { 
```

Comment:
I assume you are starting the loop at 2 to skip the header row, but it's a bit more flexible (and nicer looking) to use frozen rows to indicate the headers and then start your loop at `sheet.getFrozenRows() + 1`.

https://developers.google.com/apps-script/reference/spreadsheet/sheet#getfrozenrows

Reply:
I've done:
`var FROZEN_ROWS = sheet.getFrozenRows();`

and then in my loop:
`for (var i = FROZEN_ROWS + 1 ....`

This ok?